vignettes/decorate-module-output.Rmd

```diff
@@ -0,0 +1,589 @@
+---
+title: "Customizing Module Output"
+author: "NEST CoreDev"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  %\VignetteIndexEntry{Customizing Module Output}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+`teal` is a powerful `shiny`-based framework with built-in modules for interactive data analysis. 
+This document outlines the customization options available for modifying the output of `teal` modules. 
```

Comment:
one possible way of customization is ... a module parameter(s)
this vignette describes one of possible ways (probably the most powerful and flexible) - a decoration

we need to have a few words what decoration is (and what it is not) and then go to the details

Reply:
5d09fcf6c4e028cd66b1466dcdd906eec502f135